last run updated to save ids list instead only the last id (#35513)
demisto/content: Packs/CarbonBlackDefense/ReleaseNotes/4_0_1.md

#### Integrations

##### Carbon Black Endpoint Standard v3

> - Fixed an issue with the **fetch incidents** duplicates.

Reviewer: ```suggestion
- Fixed an issue where there was duplication with **fetch incidents**
```

> - Fixed an issue where there was duplication with **fetch incidents**.

Reviewer: ```suggestion
- fixed an issue where the integration fetched duplicate incidents.
```
